config/clouddriver.yml

```diff
@@ -40,3 +40,20 @@ google:
     - name: ${providers.google.primaryCredentials.name}
       project: ${providers.google.primaryCredentials.project}
       jsonPath: ${providers.google.primaryCredentials.jsonPath}
+
+cf:
+  enabled: ${providers.cf.enabled:false}
+
+  # TODO: Migrate this out of clouddriver and into either orca or igor
+  jenkins:
+    username: ${services.jenkins.defaultMaster.username}
+    password: ${services.jenkins.defaultMaster.password}
+
+  accounts:
+    - name: ${providers.cf.primaryCredentials.name}
+      api: ${providers.cf.primaryCredentials.api}
+      console: ${providers.cf.primaryCredentials.console}
+      org: ${providers.cf.defaultOrg}
+      space: ${providers.cf.defaultSpace}
+      username: ${cf.account.name}
```

Comment:
is it possible to make environment variables look like environment variables for clarity (UPPER_CASE)?